wdl/tasks/CallSVs.wdl

```diff
@@ -167,7 +167,7 @@ task SVIM {
         RuntimeAttr? runtime_attr_override
     }
 
-    Int disk_size = ceil(size(bam, "GB")) + 10
+    Int disk_size = 2 * ceil(size([bam, bai, ref_fasta, ref_fasta_fai], "GB"))
```

Comment:
I really wish we had a WDL function for "the size of all of the inputs provided to this task".

Reply:
There's the core, there's the standard library, some day, somewhere, there will be other popular libraries....